specification/cpim/resource-manager/readme.python.md

```diff
@@ -5,10 +5,10 @@ These settings apply only when `--track2` is specified on the command line.
 ```yaml $(track2)
 azure-arm: true
 license-header: MICROSOFT_MIT_NO_VERSION
-package-name: azure-mgmt-azureadb2c
+package-name: azure-mgmt-externalidentities
```

Comment:
It will be huge breaking change to change package name.  Add @ArcturusZhang for awareness

Reply:
It is OK to change this in track 1 since track 1 Go SDK is using multiapi.
And our track 2 package of this RP is not released yet, therefore we are OK with the new name.
But for consistency thinking, if we want to use the new name, it is better to have all the SDKs using the same name.